executor/distsql.go

```diff
@@ -277,14 +277,16 @@ func (e *IndexReaderExecutor) open(ctx context.Context, kvRanges []kv.KeyRange)
 		e.dagPB.CollectExecutionSummaries = &collExec
 	}
 
+	e.memTracker = memory.NewTracker(e.id, e.ctx.GetSessionVars().MemQuotaDistSQL)
```

Comment:
Should it use the quota of index lookup reader?

Reply:
`TableReader` and `IndexReader` have no memory quota variables now.
Actually, their work is just reading data from `DistSQL` and their trackers will be shared with `DistSQL` workers, so using `MemQuotaDistSQL` is appropriate here?

Reply:
OK